curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc8ea01436383a88256d7f.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-3
 
 # --description--
 
-The puzzle starts with the disks piled up on the first rod, in decreasing size. You need to populate your first list with numbers representing the various disk sizes.
+The puzzle starts with the disks piled up on the first rod, in decreasing size, with the smallest disk on top and the largest disk on the bottom. You need to populate your first list with numbers representing the various disk sizes.
 
-Instead of adding the items manually to the first list, generate a sequence of numbers from `3` to `1` by using the `range()` function and assign it to `rods['A']`.
+Instead of adding the items manually to the first list, generate a sequence of numbers counting down from `3` to `1` by using the `range()` function and assign it to `rods['A']`. Here, 3 represents the smallest disk at the top of the pile and 1 represents the largest disk at the bottom of the pile.
```

Comment:
This is not true: `3` represents the largest disk on the bottom and `1` the smallest one on the top. Also numbers should be surrounded with backticks.

Reply:
@Dario-DC Thank you for your feedback. Good pick up! I think I've addressed the requested changes properly now. Let me know what you think, when you have a moment.